owtf/webapp/src/containers/Dashboard/api.js

```diff
@@ -35,13 +46,25 @@ export function deleteErrorAPI(action) {
 export function getSeverityAPI() {
   const requestURL = `${API_BASE_URL}dashboard/severitypanel/`;
   // Call our request helper (see 'utils/request')
+  const options = {
+    headers: {
+      "Content-Type": "application/x-www-form-urlencoded;charset=UTF-8",
+      Authorization: "Bearer " + localStorage.getItem("token")
```

Comment:
string interpolation using backticks might be cleaner

Reply:
Changed